README.md

```diff
@@ -44,10 +40,25 @@ every time, add the following alias to your `~/.bashrc`:
 alias fluo='fluo-dev fluo'
 ```
 
+Next, you need to download the binary tarballs of Fluo dependencies (i.e Accumulo, Hadoop, and
```

Comment:
I feel like there should be a new heading in the document here, maybe a Running or Using section.  Currently this falls under the Installation heading.